src/shared/test/trim-and-dedent-test.js

```diff
@@ -17,11 +17,13 @@ describe('trimAndDedent', () => {
       
         Hello, Jane!
           Indented line
+
         Goodbye, John!
       
       `,
       `Hello, Jane!
   Indented line
+
 Goodbye, John!`,
```

Comment:
This serves the purpose to check that empty lines in between are kept, removing only those at the beginning and end.